wordpress-mobile/WordPress-Android
# Remove explore plans from Quick start

Fixes #16477 

This PR removes the Explore Plans task from the quick start list and card. 

Internal Reference: p1651666551282059/1651587009.445519-slack-C011BKNU1V5
Corresponding FluxC PR: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2409

|Before|After|  
|---|---|
|![Screenshot_20220517_183221](https://user-images.githubusercontent.com/17463767/168817152-9fb5ba83-45d2-42ab-aa8e-201580475afd.png)|![Screenshot_20220517_182531](https://user-images.githubusercontent.com/17463767/168817134-90c72035-c273-4baa-856d-cd960127e9c4.png)| 
---

### To test:

#### Test 1
Explore Plans is Not Shown on Quickstart list 

1.  Login to the app with a quick start in progress
2. Click on My Site -> Next Steps Card
3. Notice that Explore Plans quick start task is not displayed

#### Test 2 - Migration 
Scenario - Test the case where all Quickstart tasks other than Explore Plan is completed, In this task completed notification should be shown to the user in Notifications tab 

Pre- requisites 
For this test, install an app having the `Explore Plans` listed, 

1.  Login to the app with a quick start in progress
2. Complete all tasks other than Explore Plans 
3. Checkout to this branch or Install the build from this PR
4. Notice that Explore Plans is not displayed and Quick start is completed 


---
### Merge Instructions
1. Make sure the dependent FluxC PR is merged 
2. Update FluxC hash
3. Remove Not Ready for Merge 

## Regression Notes
1. Potential unintended areas of impact
- Quickstart progress not shown properly 
- Explore plans tasks being shown 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing 

3. What automated tests I added (or what prevented me from doing so)
N/A

---
PR submission checklist:
